packages/p/src/p.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+
+type PElement = React.ElementRef<'p'>;
+type RootProps = React.ComponentPropsWithoutRef<'p'>;
+
+export interface PProps extends RootProps {}
+
+export const P = React.forwardRef<PElement, Readonly<PProps>>(
+  ({ style, ...props }, forwardedRef) => (
+    <p
+      ref={forwardedRef}
+      style={{
+        color: "#000",
```

Comment:
I think the default should be inherited from parent.

```suggestion
```

Reply:
great suggestion